mithril.d.ts

```diff
@@ -851,8 +851,4 @@ declare namespace Mithril {
 	}
 }
 
-declare const m: Mithril.Static;
-
-declare module "mithril" {
-    export = m;
-}
+export const m: Mithril.Static;
```

Comment:
Only this export work, if create default export, then 
```typescript
import {m, Mithril} from 'mithril';
```
If add default export, then him block all others export.
In js work both variants.